examples/kcp-certs/run.bash

```
@@ -259,66 +259,23 @@ _stop_broker() {
-        secret_try_count=$((secret_try_count + 1))
-        sleep 1
-    done
-    die "Timed out waiting for certificate to be issued with expected fqdn '$expected_fqdn'"
+    kubectl::wait::suffix "$kubeconfig" "$resource" default \
+        ".metadata.annotations.broker\.platform-mesh\.io/provider-cluster" \
+        "$expected_cluster_id"
+
+    kubectl::wait "$kubeconfig" "certificates/cert-from-consumer" default \
+        jsonpath='{.status.relatedResources.secret.name}'
+
+    local secret_name="$(kubectl --kubeconfig "$kubeconfig" get certificates cert-from-consumer -o jsonpath='{.status.relatedResources.secret.name}')"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)